dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/zkclient/ZkclientZookeeperClient.java

```diff
@@ -37,7 +37,8 @@
 
     public ZkclientZookeeperClient(URL url) {
         super(url);
-        client = new ZkClientWrapper(url.getBackupAddress(), 30000);
+        long timeout = url.getParameter("timeout", 30000L);

```

Comment:
also the key "timeout".
:)

Reply:
make sense. this one can replace by `TIMEOUT_KEY`